submissions/reactjs/assignment2.MD

```diff
@@ -23,7 +23,7 @@
 | Sparsh Patel                   | 2203031080053        |                   |             |
 | Pratik Patel                   | 2203031080174        |                   |             |
 | Ravi Vira                      | 2203031080018        |                   |             |
-| Deep Menpara                   | 2203031080091        |                   |             |
+| Deep Menpara                   | 2203031080091        |https://github.com/Deep7133/ReactJS/blob/main/Frontend/components/Greet.jsx                   |https://github.com/Deep7133/ReactJS/tree/main/Frontend             |
```

Comment:
Please follow the repository structure mentioned in the problem statement document